homework_11/test_log_event.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from homework_11.log_event import log_event
+
+
+def test_log_success(caplog):
```

Comment:
If you use pytest framework - you need to store your test cases methods inside Test class.